release_testing/dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build docker image with which to test pkgnet on r-devel debian
+# intended build command to be run from package root:
+# 		docker build -f release_testing\dockerfile -t pkgnet_rdevel .
+
+FROM rocker/r-devel
+
+# Copy test script
+COPY release_testing/r_devel_check.sh .
+RUN chmod +x r_devel_check.sh
+
+# Copy pkg in order to install packages
+COPY . pkgnet/
+
+# RUN apt-get update && \
+RUN apt-get install -y pandoc pandoc-citeproc qpdf ghostscript libssl-dev libxml2-dev
+
+RUN RD -e 'install.packages(c("xml2", "rversions", "roxygen2", "devtools"))'
```

Comment:
I think you need to set `repos` here, don't you? Or do the `rocker` images come with a built-in `~/.Rprofile` that has that set in it

Reply:
It worked without it, but this is a good practice. Adding it.